src/plugins/data/common/search/aggs/param_types/field.ts

```diff
@@ -84,16 +84,17 @@ export class FieldParamType extends BaseParamType {
       }
 
       const validField = this.getAvailableFields(aggConfig).find((f: any) => f.name === fieldName);
-      if (!validField) {
+      if (!validField || 1 === 1) {
```

Comment:
should this not just be `true`? :D

Reply:
:D committed my debug for demonstration code